docs/configuration/coordinates.rst

```diff
@@ -0,0 +1,84 @@
+## Coordinates in Navit
```

Comment:
I just happen now to see that you used markdown instead of reStructuredText which results in no headline found [here](https://navit--1219.org.readthedocs.build/en/1219/configuration/general.html#initial-map-position) and not format [here](https://navit--1219.org.readthedocs.build/en/1219/configuration/coordinates.html)

Reply:
Just updated coordinates.rst to be in rst format (at least I choose export reStructuredText in Typora to create it). Added some lines for Google Maps coordinates as well.